src/components/app.tsx

```diff
@@ -235,6 +236,15 @@ export class App extends React.PureComponent<IProps, IState> {
                                  ? activityIndex + 1
                                  : undefined;
     sequenceActivity !== undefined && setQueryValue("sequenceActivity", sequenceActivity);
+    const backgroundImage = sequence?.background_image
+                              ? sequence.background_image
+                              : activity.background_image
+                                ? activity.background_image
+                                : undefined;
```

Comment:
You can collapse all of this into a single line if you also make the change on the line below.  The above only needs to be `const backgroundImage = sequence?.background_image || activity.background_image;`

Reply:
Thanks for the great suggestions. I've made all these changes.